tests/test_import_file.py

```diff
@@ -0,0 +1,176 @@
+import datetime
+
+import pytest
+
+from app.internal.import_file import (
+    before_import_checking, check_date_in_range, check_file_extension,
+    check_file_size, check_validity_of_txt, import_ics_file, import_txt_file,
+    is_file_exist, user_click_import)
+
+
+file_size_tests = [
+    # File size above the constraint we set in the global variable (5 MB)
+    (r"tests/files_for_import_file_tests/sample_above_5mb.txt", False),
```

Comment:
You are using the same file multiple times in different tests. Move it to a constant variable so if any changes need to happen to the file name, you'll only need one place to edit this.

Reply:
Done